src/main/java/org/broadinstitute/hellbender/utils/MannWhitneyU.java

```diff
@@ -534,7 +534,12 @@ public double permutationTest(final double[] series1, final double[] series2, fi
          * and dividing by the total count of everything in the histogram. Just using getCumulativeDistribution() gives
          * a p-value of 1 in the most extreme case which doesn't result in a usable z-score.
          */
-        double sumOfAllSmallerBins = histo.get(testStatU).getValue() / 2.0;
+        double sumOfAllSmallerBins;
+        if ( histo.get(testStatU) == null ) {
```

Comment:
Can you save the return value from the first `histo.get(testStatU)` call so that you don't have to call it a second time below?